doc/configuration.md

```diff
@@ -107,3 +107,27 @@ You may configure the logging level by using the `LogLevel` option in your CNI c
     "LogLevel": "debug",
 ```
 
+### Specify default cluster network in Pod annotations
+
+With Multus, user can also specify the default cluster network for a Pod when there are more than one cluster network deployed for a Kubernetes cluster.
+
+The possible use cases could be, for example:
+
+1. For cluster network migration (e.g. from flannel to calico), it would be nice if both network solutions can work in parallel. User can control which network a pod should attach to during the transition period.
+1. Some user may deploy multiple cluster networks for sake of security, and want to be able to specify cluster network for pods.
+
+Here comes the steps to use this feature:
+
+1. First, you need to define all your cluster networks as network-attachment-definition objects.
+
+2. Then you can specify the network you want in pod annotation 'v1.multus-cni.io/default-network'. Pods without specifying the annotation will keep using the CNI defined in multus config file.
```

Comment:
2. Next, you can specify the network you want in pods with the `v1.multus-cni.io/default-network` annotation. Pods which do not specify this annotation will keep using the CNI as defined in the Multus config file.